CONTRIBUTING.rst

```diff
@@ -143,19 +140,25 @@ and how to set up your development environment as introduced in the Zephyr
 .. _Getting Started Guide:
    https://www.zephyrproject.org/doc/getting_started/getting_started.html
 
-The examples below use a Linux host environment for Zephyr development.
 You should be familiar with common developer tools such as Git and CMake, and
 platforms such as GitHub.
 
 If you haven't already done so, you'll need to create a (free) GitHub account
 on http://github.com and have Git tools available on your development system.
 
+.. note::
+   The Zephyr development workflow supports all 3 major operating systems
+   (Linux, macOS and Windows) but some of the tools used in the sections below
+   are only available on Linux and macOS. On Windows you will need to rely on
```

Comment:
How about:

    On Windows, instead of running these tools yourself, you will need to rely on
    the Continuous Integration (CI) service ``shippable``, which runs automatically
    on GitHub when you submit your Pull Request (PR).  You can see any failure
    results in the Shippable details link near the end of the PR conversation list.

(As an aside, issue #4772 mentions we don't document how to find and interpret these shippable reports.)